Statistics/Distributions/ShiftedGamma.cs

```diff
@@ -15,7 +15,7 @@ internal ShiftedGamma(double alpha, double beta, double shift)
             // The Gamma distribution is defined by 2 parameters: 
             //      (1) alpha is the shape parameter 
             //      (2) beta is the rate or inverse scale parameter
-            _Gamma = new Gamma(alpha, 1/beta);
+            _Gamma = new Gamma(alpha, beta); //Gamma(alpha, 1/beta);
```

Comment:
good catch. fix the comment. beta is not the inverse scale, it is the scale (if we dont divide by 1?).